webserver/templates/macros.html

```diff
@@ -13,38 +13,37 @@
 {% endmacro %}
 
 {% macro pager(links, prevpage, current_page, last_page, nextpage) %}
-	<nav>
-	  <ul class="pagination">
-		{% if current_page == 1 %}
-		<li class="disabled">
-        <span aria-hidden="true">&laquo;</span>
-		{% else %}
-		<li>
-          <a href="{{prevpage}}" aria-label="Previous">
-			<span aria-hidden="true">&laquo;</span>
-		  </a>
-		{% endif %}
-		</li>
-		{% for number, link in links %}
-			{% if current_page == number %}
-			<li class="active">
-			{% else %}
+    <nav>
+        <ul class="pagination">
+            {% if current_page == 1 %}
+            <li class="disabled">
+                <span aria-hidden="true">&laquo;</span>
+            {% else %}
             <li>
+                <a href="{{prevpage}}" aria-label="Previous">
+                    <span aria-hidden="true">&laquo;</span>
+                </a>
+            {% endif %}
+            </li>
+            {% for number, link in links %}
+                {% if current_page == number %}
+                <li class="active">
+                {% else %}
+                <li>
```

Comment:
This is a bit funny. If I use
    <li {{'class=disabled' if current_page == 1}}>
then it works, since the browser correctly parses the class even though there are no `"` quotes around the class name.
Adding quotes:
    <li {{'class="disabled"' if current_page == 1}}>
results in jinja escaping them, so that the resulting source is 
    <li class=&#34;disabled&#34;>
which isn't really right either. 

My preference is for quotes around attributes when possible, and so I think I'll leave it as is. Adding a `|safe` filter seems just as messy.